src/views/Home.vue

```diff
@@ -166,7 +166,7 @@
               </span>
             </router-link>
           </div>
-          <div v-if="isElectron() && !isMac()" class="window-control">
+          <div v-if="isElectron() && isLinux()" class="window-control">
```

Comment:
test in windows 10, needs window buttons to control.

Reply:
Maybe should test again with `titleBarOverlay: true` I ran into the same error due to empty titleStyle

Reply:
`titleBarOverlay` is nice in windows env to show styled  window control, but one drawback is user when user change theme, I find no way to change titleBarOverlay without recreate browser window. So I think if there's no way to do that, we can't use titleBarOverlay, should draw window control by ourself.

Reply:
I think a prompt for recreate or restart would be sufficed because I really think user would be more comfortable dealing with navbars that fit their habit and I would say the drawback on switch themes is acceptable. Users have to restart to change front or background modes in chrome extension v2.

Reply:
Actually, we can make the color transparent and make some hack. https://github.com/microsoft/vscode/issues/127449#issuecomment-958631549
However, it still looks buggy right now. Only maximized seem to work good on my setup. Maybe we can wait until it was fixed by upstream.

Reply:
adjust to fit windows style:
https://github.com/listen1/listen1_chrome_extension/pull/742